public/stylesheets/site/2.0/18-zone-searchbrowse.css

```diff
@@ -162,4 +152,66 @@ form.filters {
   content: none;
 }
 
+.filters .sort + .heading {
+  margin-top: 1.286em; /* heading top margin minus sort bottom margin */
+}
+
+/* INTERACTION: OLD FILTERS */
+/* ES UPGRADE TRANSITION: REMOVE WHEN DONE */
+
+form.old-filters {
```

Comment:
`old-filters` 100% violates our class naming conventions, but this is definitely temporary, so I figure we can live with it.